src/test/shared/utilities/childProcess.test.ts

```diff
@@ -25,163 +25,11 @@ describe('ChildProcess', async function () {
         await tryRemoveFolder(tempFolder)
     })
 
-    describe('run', async function () {
```

Comment:
looks like these were duplicates, correct? it is helpful to mention that in the commit message

Reply:
Ah yeah they test the same thing in a different way.